### PR TITLE
Drop support for OTEL_DOTNET_AUTO_[SIGNAL]_CONSOLE_EXPORTER_ENABLED env. vars.

### PR DESCRIPTION
**Pull request by ysolomchenko**
Fixes https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/3540

## What

Drop support for following env. vars.:

- OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED
- OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED
- OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED 

## Checklist

- [x] `CHANGELOG.md` is updated.
- [x] Documentation is updated.
- [ ] New features are covered by tests.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -34,13 +34,6 @@ or use the appropriate install script:
 > but they can't be used in all cases. See [Limitations](./using-the-nuget-packages.md#limitations)
 > for details.
 
-To see the telemetry from your application directly on the standard output, set
-the following environment variables to `true` before launching your application:
-
-- `OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED`
-- `OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED`
-- `OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`
-
```
Comment (Kielek) [inline docs/README.md:43]: Could you please describe the new approach here?
Debugging purposes seems to be important feature at the beginning of the journey with OTel .NET Auto.

File: test/test-applications/integrations/TestApplication.NServiceBus/Properties/launchSettings.json [hunk 2]
```diff
@@ -15,14 +15,12 @@
...
         "OTEL_LOG_LEVEL": "debug",
-        "OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED": "true",
         "OTEL_DOTNET_AUTO_HOME": "$(SolutionDir)bin\\tracer-home\\",
         "OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_SOURCES": "TestApplication.*",
         "OTEL_LOGS_EXPORTER": "none",
         "OTEL_LOG_LEVEL": "debug",
         "OTEL_METRICS_EXPORTER": "otlp",
         "OTEL_METRIC_EXPORT_INTERVAL": "100",
-        "OTEL_TRACES_EXPORTER": "none"
       }
```
Comment (Kielek) [inline test/test-applications/integrations/TestApplication.NServiceBus/Properties/launchSettings.json:24]: `"OTEL_TRACES_EXPORTER": "console"`?
Consider similar changes for other  json files.

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +17,11 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ### Removed
 
+Support for following environment variables:
+`OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`,
+`OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED`, and
+`OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED`.
```
Comment (pellared) [inline CHANGELOG.md:23]: Missing bullet. Also it would be good to describe what users can use instead.

File: docs/README.md (hunk 1 above)
Comment (ysolomchenko) [reply docs/README.md]: fixed in 984a940f0808cf6b22c83ef6258836f9eed2549a

File: test/test-applications/integrations/TestApplication.NServiceBus/Properties/launchSettings.json (hunk 2 above)
Comment (ysolomchenko) [reply test/test-applications/integrations/TestApplication.NServiceBus/Properties/launchSettings.json]: fixed in 984a940f0808cf6b22c83ef6258836f9eed2549a

**Review by pjanotti — APPROVED** [review]

**Review by rajkumar-rangaraj — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,6 +17,15 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
...
 ### Removed
 
+- Support for following environment variables:
+  `OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`,
+  `OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED`, and
+  `OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED`.
+  Use `console` as one of the values for the following environment variables:
+  `OTEL_TRACES_EXPORTER`
+  `OTEL_METRICS_EXPORTER`
+  `OTEL_LOGS_EXPORTER`
```
Comment (pellared) [inline CHANGELOG.md:27]:
```suggestion
- Drop support for following environment variables:
  `OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`,
  `OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED`, and
  `OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED`.
  Instead, use `console` as one of the values for the following environment variables:
  `OTEL_TRACES_EXPORTER`
  `OTEL_METRICS_EXPORTER`
  `OTEL_LOGS_EXPORTER`.
```

**Review by pellared — APPROVED** [review]

**Pull request merged**